### ISSUE TITLE
Cura frequently stops auto-reslicing after performing an action that would initiate a reslice.

### ISSUE THREAD

**Issue by aviphysics** [Ultimaker/Cura#260]
Cura frequently stops auto-reslicing after performing an action that would initiate a reslice. Cura deletes the slices but does not apparently start a new slice, or sometimes freezes part way through the reslice.

Observed in all 15.06 versions running on Windows 7.

**Comment by nallath** [comment]
Could you please have a look at https://github.com/Ultimaker/Cura#logging-issues 
We require more information before we can help you.

**Comment by nallath** [comment]
The delete now triggers a reslice. Please create a new issue when other interactions do not trigger it.
